app/Template/board/task_footer.php

```diff
@@ -68,7 +68,11 @@
                 <?php endif ?>
                 ">
                 <i class="fa fa-calendar"></i>
-                <?= $this->dt->datetime($task['date_due']) ?>
+                <?php if (date('H', $task['date_due']) > 0 || date('i', $task['date_due']) > 0 ): ?>
```

Comment:
Can we simplify this condition? for example: `date('Hi', time()) === '0000'`

Reply:
seems to be much better, I've changed it :)